### PR TITLE
Add PHP Signed Server Request example

### PR DESCRIPTION
**Pull request by jmealo**
This PR includes a PHP class to generate a signed server request and an example of how to use it (same example as Node.js with some HTML removed for added clarity).

I tested the output against the Node.js example and given the same ``token``, ``client_id`` and ``app_secret`` it generated identical output.

### CODE REVIEW

File: signing_server_examples/php/generate_signed_request.php [hunk 1]
```diff
@@ -0,0 +1,61 @@
+<?php
+
+require('OpenEd/SignedServerRequest.php');
```
Comment (jmealo) [inline signing_server_examples/php/generate_signed_request.php]: @themightychris: For portabilities sake of the example I should leave this as ``require`` correct? If they're using an autoloader that's PSR compliant they'd want to do ``use``.

Comment (LeonidMorozov) [inline signing_server_examples/php/generate_signed_request.php]: I think we can use `require_once` here to be sure there's no issues.

File: signing_server_examples/php/OpenEd/SignedServerRequest.php [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+namespace OpenEd;
+
+
+class SignedServerRequest
+{
+    private $app_secret;
+    private $client_id;
+
+    public function __construct($app_secret, $client_id)
```
Comment (LeonidMorozov) [inline signing_server_examples/php/OpenEd/SignedServerRequest.php]: please, move `client_id` to the 1st place

File: signing_server_examples/php/OpenEd/SignedServerRequest.php [hunk 3]
```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace OpenEd;
+
+
+class SignedServerRequest
+{
+    private $app_secret;
```
Comment (LeonidMorozov) [inline signing_server_examples/php/OpenEd/SignedServerRequest.php]: please, rename it to `client_secret` and push it below the `client_id`.

**Comment by LeonidMorozov** [comment]
@jmealo , thank you for PR. Good job.
Please, fix small comments and I'll merge it.

**Comment by jmealo** [comment]
@LeonidMorozov Thank you. I believe commit b12942b addresses your feedback. I also changed ``app_secret`` to ``client_secret`` in the Node.js example for consistency.

File: signing_server_examples/php/generate_signed_request.php (hunk 1 above)
Comment (themightychris) [inline signing_server_examples/php/generate_signed_request.php]: Someone using an autoloader will be reorganizing this anyway. `use` has nothing to do with class loading, only namespace handling.

**Pull request merged**